test-suite/mock-spine-mhs-outbound/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
-    id 'org.springframework.boot' version '2.7.1'
-    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-    id 'io.freefair.lombok' version '6.2.0'
+    id 'org.springframework.boot' version '3.3.3'
+    id 'io.spring.dependency-management' version '1.1.6'
```

Comment:
Same as the comment above, `1.1` should suffice - if we choose that this is OK to do so.

Reply:
again, I am aligning it with the other modules, like gp2gp-translator or common or db-connector... they are all using ver. 1.1.6. I think the general rule is that it is better to use more specific version given we are already have other modules using the same version.

Reply:
Okay doak! Makes sense, just thought I'd highlight!